code/game/objects/items/weapons/cash.dm

```diff
@@ -43,14 +43,15 @@ var/global/list/moneytypes=list(
 	else
 		setGender(NEUTER)
 		..()
+		user << "<span class='info'>It's worth [worth] credit.</span>"
```

Comment:
This still isn't necessary, its already being described as a singular credit of some amount. There's no need to say, this is a 100 credit chip, it's worth 100 credit.